### DISCUSSION TITLE
Rate Limiting/Crashing Plex?

### DISCUSSION THREAD

**Discussion by faultoverload** [meisnate12/Plex-Meta-Manager#472] [Q&A]
Has anyone had issues with Plex crashing during initial runs of collection building? I had it setup to run through a few of the collection builds in meisnate12's folder but after a time I just started getting `Unknown Error: HTTPConnectionPool(host='10.0.1.20', port=32400): Read timed out. (read timeout=30) ` error messages from the console and Plex started dropping streams and took a while to recover. Is there anything I can do to stop this from happening short of just cutting out those collection build files?

**Comment by meisnate12** [comment]
you can try changing the plex timeout in the config but tbh sometimes plex just takes forever to respond and I'm not sure why
